k8s/grafana/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,22 @@
 actions:
-- name: dummy test
+- name: grafana listening
   bashTest:
     script: |-
-      exit 0
+      sleep 120
+      curl http://$APP_INSTANCE_NAME-grafana.$NAMESPACE.svc.cluster.local:3000
     expect:
-      exitCode:
-        equals: 0
+      stdout:
+        contains: 'login'
+- name: graphana login
```

Comment:
Can you add negative test as well? `Can not login to Grafana with incorrect credentials`